CMakeLists.txt

```diff
@@ -1,6 +1,15 @@
 cmake_minimum_required( VERSION 3.0 FATAL_ERROR )
 set( CMAKE_VERBOSE_MAKEFILE ON )
 
+# Configure the Android toolchain before the project start
+if( CINDER_TARGET )
```

Comment:
I seems that the proper ( in terms of CMake ) way to do this would be using CMake's toolchain functionality. See [here](http://www.vtk.org/Wiki/CMake_Cross_Compiling) and [here](http://www.vtk.org/Wiki/CMake_Cross_Compiling#The_toolchain_file). This functionality would also be interesting to explore in the case of Linux / RPi .